### PR TITLE
add a Bug Report issue template to GH

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
Does what it says on the tin (hopefully).

Resolves https://github.com/okta/odyssey/issues/99

### CODE REVIEW

**Comment by johnintrater-okta** [comment]
I have no idea how to view the template. Can you point me to the right direction? Sorry

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+name: "Bug Report"
+about: "Oops, all berries? Let us know."
```
[the remark this answers is not in the document]
Comment (edburyenegren-okta) [reply .github/ISSUE_TEMPLATE/bug-report.md]: *I suppose.*

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+title: ''
+labels: "bug"
+assignees: ''
+---
+<!-- Please title this ticket with a brief summary of your issue, then answer the questions below. -->
+
+## Detailed description
+
```
[the remark this answers is not in the document]
Comment (edburyenegren-okta) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Is this something we'll want generally? Ideally, we won't be investigating feature-specific implementations within the product.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+## Steps to reproduce the issue
+
+1. Step one.
+2. Step two.
+3. ???
+4. Profit.
+
+## Screenshots
+
+Please include screenshots to help illuminate the issue.
```
[the remark this answers is not in the document]
Comment (edburyenegren-okta) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Nope :(

**Comment by edburyenegren-okta** [comment]
Mergin now for QA session, we can iterate as we go.

**Pull request merged**